torch_geometric/nn/conv/message_passing.py

```diff
@@ -466,26 +469,20 @@ def explain_message(self, inputs: Tensor, size_i: int) -> Tensor:
         return inputs * edge_mask.view(size)
 
     def aggregate(self, inputs: Tensor, index: Tensor,
-                  ptr: Optional[Tensor] = None, dim_size: Optional[int] = None,
-                  aggr: Optional[str] = None) -> Tensor:
```

Comment:
any chance we'll be breaking code by doing this? Seems like a class people may already be building ontop of?

Reply:
`aggr` was just introduced in master and now we are removing it. I think we are safe.